HasDropDown.js

```diff
@@ -225,7 +225,7 @@ define([
 		postRender: function () {
 			this.behaviorNode = this.behaviorNode || this;
 			this.buttonNode = this.buttonNode || this.behaviorNode;
-			this.popupStateNode = this.focusNode || this.buttonNode;
+			this.popupStateNode = this.popupNode || this.focusNode || this.buttonNode;
```

Comment:
Oh I didn't realize you couldn't specify `popupStateNode` directly.  I think `popupStateNode` should be directly settable, just like `behaviorNode` and `buttonNode`.